File: python/RemoveDuplicatesfromSortedArray.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums:
            return 0
        top = 0
        for n in nums:
            if n != nums[top]:
                top += 1
                nums[top] = n
        return top + 1

File: C/RemoveDuplicatesfromSortedArray.c

int removeDuplicates(int* nums, int numsSize)
{
	if (numsSize == 0)
		return 0;
	int top = 0;
	for (int i = 0; i < numsSize; ++i)
	{
		if (nums[i] != nums[top])
			nums[++top] = nums[i];
	}
	return top + 1;
}
